src/main/java/net/moonbyte/AnarchyUtils/Commands/HelpCommand.java

```diff
@@ -9,12 +9,13 @@
 import org.bukkit.entity.Player;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.Collections;
 import java.util.List;
 
 public class HelpCommand implements CommandExecutor {
     @Override
     public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command command, @NotNull String s, @NotNull String[] strings) {
-        final List<String> list = ConfigurationUtils.getConfig().getStringList("global.help-lines");
+        final List<String> list = Collections.singletonList(ConfigUtil.Global_HelpLines);
```

Comment:
I'm not 100% sure if this would work, and since other commands were not tested, can you post a Screen Shot of the help command or verify that the help command correctly works?